### DISCUSSION TITLE
Translations for Documentation PRs

### DISCUSSION THREAD

**Discussion by kwill39** [rrousselGit/riverpod#2529] [Q&A] [answered]
Should all documentation pull requests have other translations as well? It looks like Riverpod documentation supports multiple languages. I've opened a few PRs for documentation already, but I didn't think about making sure the new documentation was available in multiple languages.

**Comment by rrousselGit** [comment, marked as answer]
No.

Instead when the content of the EN page is changed in a "breaking" way, the page's version should be bumped.
This will show an "out of date" banner on older translations until they update.

An example is:

https://github.com/rrousselGit/riverpod/edit/docs-v2/website/docs/getting_started.mdx

You can see the `version: 2`.

**Reply by kwill39** [reply]
Just so I'm clear, what might classify as a "breaking" change when it comes to documentation?

Obviously, simple punctuation or grammatical changes would not be considered breaking. But if someone adds an Info/Danger block to a page, is that considered "breaking" because it's new content?

**Reply by rrousselGit** [reply]
Yes. Anything that would have to be reported on translated pages should involve a version bump.
